test/onnx-model-zoo/CheckONNXModelZoo.py

```diff
@@ -126,18 +156,16 @@ def obtain_all_model_paths():
     model_names = [
         path.split('/')[-1][:-len(".tag.gz")] for path in model_paths
     ]  # remove .tag.gz
-    deprecated_names = set(model_names).intersection(deprecated_models)
+    excluded_names = set(model_names).intersection(excluded_models)
 
     log_l1('\n')
-    deprecated_msg = ""
-    if (len(deprecated_names) != 0):
-        deprecated_msg = "where " + \
-            str(len(deprecated_names)) + \
-            " models are deprecated (using very old opsets, e.g. <= 3)"
-    log_l1("# There are {} models in the ONNX model zoo {}".format(
-        len(model_paths), deprecated_msg))
-    log_l1("See https://github.com/onnx/models/pull/389",
-           "for a list of deprecated models\n")
+    excluded_msg = ""
+    if (len(excluded_names) != 0):
+        excluded_msg = "where " + \
+            str(len(excluded_names)) + \
+            " models are not checked because of old opsets or quantization"
```

Comment:
Instead of line continuation, it's probably better to use parentheses for multi-line expressions:
```
excluded_msg = ("where " +
    str(len(excluded_names)) +
    " models are not checked because of old opsets or quantization")
```

Reply:
Thanks for reminding this! It's better.